VKApp/VKApp/Controllers/GroupUserTableViewController.swift

```diff
@@ -78,26 +73,55 @@ final class GroupUserTableViewController: UITableViewController {
             let source = segue.source as? SearchGroupTableViewController,
             let indexPath = source.tableView.indexPathForSelectedRow,
             let group = source.returnGroup(index: indexPath.row),
-            !userGroups.contains(group)
+            !vkGroups.contains(group)
         else { return }
-        for (index, groupFromAllGroups) in allGroups.enumerated() {
-            guard group == groupFromAllGroups else { continue }
-            allGroups.remove(at: index)
-        }
-        userGroups.append(group)
-        tableView.reloadData()
+        vkGroups.append(group)
+        realmService.saveGroupVKData(vkGroups)
     }
 
     // MARK: - Private Methods
 
     private func setupView() {
-        vkNetworkService.fetchUserGroupsVK { [weak self] items in
+        setupNotificationToken()
+        loadFromRealm()
+        loadFromNetwork()
+    }
```

Comment:
```suggestion
        loadData()
    }
```
 а внутри уже вызывай нужные методы